site/src/components/WorkspaceScheduleForm/WorkspaceScheduleForm.tsx

```diff
@@ -40,8 +40,8 @@ export const Language = {
   startTimeLabel: "Start time",
   startTimeHelperText: "Your workspace will automatically start at this time.",
   timezoneLabel: "Timezone",
-  ttlLabel: "TTL (hours)",
-  ttlHelperText: "Your workspace will automatically shutdown after the TTL.",
+  ttlLabel: "Time until shutdown (hours)",
```

Comment:
What do you think about `Hours until shutdown`?

Reply:
There's a comment on the issue that talks about eventually having minutes/seconds resolution as well: https://github.com/coder/coder/issues/1784#issuecomment-1138973156.